src/fidesops/graph/data_type.py

```diff
@@ -107,8 +107,8 @@ class BooleanTypeConverter(DataTypeConverter[bool]):
     def __init__(self) -> None:
         super().__init__(name="boolean", empty_val=False)
 
-    true_vals = {"True", "true", True, 1}
-    false_vals = {"False", "false", False, 0}
+    true_vals = ("True", "true", True, 1)
```

Comment:
I changed to a tuple because `True` and `1` / `False` and `0` are the same thing behind the scenes so in a set `{"True", "true", True, 1}` you get `{"True", "true", True}`. I change it to a tuple to not drop the number.

Reply:
nice, good find!